docs/advanced/retries.rst

```diff
@@ -66,12 +66,12 @@ times before giving up:
 
     @attrs.define
     class MyPage(WebPage):
-        http_client: HttpClient
+        http: HttpClient
 
         @retry(stop=stop_after_attempt(3))
         async def get_data(self):
             request = HttpRequest("https://toscrape.com/")
-            response = await self.http_client.execute(request)
+            response = await self.http.execute(request)
```

Comment:
Hm, in case of .execute and .batch_execute, `http_client` looks slightly better than `http`.